go/cmd/vtctldclient/command/workflows.go

```diff
@@ -173,6 +184,6 @@ func init() {
 	WorkflowUpdate.MarkFlagRequired("workflow")
 	WorkflowUpdate.Flags().StringSliceVarP(&workflowUpdateOptions.Cells, "cells", "c", []string{}, "New Cell(s) or CellAlias(es) (comma-separated) to replicate from")
```

Comment:
Since I'm running the CI tests again for this little improvement: https://github.com/vitessio/vitess/pull/12622/commits/1cbb4c5d257d009b8f1c4222988c9c3abb4e429f

I went ahead and made this change (could see that we're doing this for other StringSlice flags in vtctldclient code): https://github.com/vitessio/vitess/pull/12622/commits/12eb98ce9f18365c617e06b9910b84b3edc19d29